Report/Chapters/Requirements.tex

```diff
@@ -1,26 +1,76 @@
 
-\subsection{User}
+To start however, we needed something more than just an idea. We needed a base from which we could work. This resulted in a set of requirements, which we defined based on the ideas of our client.
 
-How to divide the requirements in necessary and desirable requirements depends on the kind of target user for which the application is to be designed. 
-In this project the focus lies mainly on users who already know how to programme in Haskell. %nog aanvulling nodig,..
+\section{Target audience}
 
-\subsection{Requirements}
-For this project, the requirements can be devided in three subgroups, namely: visual requirements, functional requirements and multi-touch requirements. In the following subsections each of the groups will be discussed. In some cases a requirement fits in multiple groups; in these cases the best fitting group has been chosen to contain this requirement.
+How to divide the requirements in necessary and desirable requirements depends on the kind of target audience for which the application is designed. \index{audience} \index{target audience}
 
+In this project the focus lies mainly on users who already know how to program in Haskell. This includes people who are just starting with Haskell.
+The program should aid these users in creating an overview of their program and informing them about (possible) mistakes without the user having to write a complete program and then compile it.
 
-\subsubsection{Visual requirements} 
+Furthermore, the users are generally advanced computer users and therefore do not require an in-depth explanation on how to operate a program. The users will also have a basic understanding of touch gestures, most likely obtained through the use of a smart phone or tablet.
 
-As discussed earlier, the user experience is a very important aspect in this project. With this in mind, the visual aspects of the application are important. De visual requirements are the following:
+\section{Hardware}
+
+The program will be designed to use on a large multi-touch screen with a screen resolution of 1920x1080 pixels, but should also work with a smaller (for example laptop) screen and a mouse. The program should be efficient enough to run on average modern-day hardware. The client provided a desktop computer and multi-touch screen for reference.
+
+\section{Requirements}
+
+For this project, the requirements can be divided in three subgroups, namely: visual requirements, functional requirements and multi-touch requirements. In the following subsections each of the groups will be discussed. In some cases a requirement fits in multiple groups; in these cases the best fitting group has been chosen to contain this requirement.
```

Comment:
Why? This is just a split to put some things into context, not because things only apply to the subgroup they're in.